### DISCUSSION TITLE
What are the tools for viewing telegram bot user statistics?

### DISCUSSION THREAD

**Discussion by jekeam** [python-telegram-bot/python-telegram-bot#3236] [Q&A]
I would like to see statistics in the bot similar to those for channel owners, described here: https://core.telegram.org/api/stats .

What are the tools/services for chat bots in more detail?

**Comment by Poolitzer** [comment]
hey, that documentation is for the main or MTProto api, not the http bot api this wrapper is for. Check out the documentation for it at https://core.telegram.org/bots/api

**Reply by jekeam** [reply]
I think you misunderstood my question. I'm using the python-telegram-bot library and I need the community's help to understand what tools exist to get statistics about the usage of my telegram bot in the context of your library. Does everyone write their decisions? Can anyone share their work? I would like to do simple actions and create something similar on my bot - see screenshots. I know that the official api telegram does not provide any statistics on the bot.
<p><img src="https://user-images.githubusercontent.com/19660852/190036837-87e2efd4-875f-4ea5-a6e3-48d601bef2e4.jpg" width="250"/>
<img src="https://user-images.githubusercontent.com/19660852/190036880-e02ba6c0-30a6-432d-a136-c805e944a113.jpg" width="250"/>
<img src="https://user-images.githubusercontent.com/19660852/190037438-f7eb796e-ce88-4340-bfcd-e9926f1945e4.jpg" width="250"/>
</p>
